README.md

```diff
@@ -15,9 +15,53 @@ Check the [issues](http://github.com/Chewsterchew/DIRCord/issues) to see what I'
 
 ## Setup guide.
 
-1) Create a server on Discord.
+1) Create a server on Discord, copy the *server's ID*, you'll need it for later.
 
-2) more soon lol
+2) Copy your *user ID*, you'll need it for later.
+
+3) Go to the [Discord Developer Portal](https://discordapp.com/developers/applications/) Make a new application, give it a name and an avatar if you want.  Copy the *Client ID*, you'll need it for later.  Then head to the **Bot** Section.  Create a bot, scroll down and select the **Administrator** Permission.  Make sure you unselect the slider that makes it a public bot, you don't want anyone but you to have this bot.  Hit the **Copy** button underneath where it says *Bot token*, you'll need this for later.
+
+4) Invite the bot to the server, if you don't know how to do that, go [here](https://discordapi.com/permissions.html)  Click on the **Administrator** permission box, and scroll down.  Paste in your *Client ID* that you got from the Discord Developer Portal, and open the link at the bottom of the page.  **DO NOT CHECK ANY OTHER BOXES OTHER OR FILL IN ANYTHING ELSE OTHER THAN THE CLIENT ID.**
+
+5) In the invite link, select your server that you made in Step 1, then invite the bot.
+
+## Here is where the fun begins: *These steps should be universal, but might not work on all devices, these instructions were made for linux, specifically Ubuntu.*
+
+6) Open up your CLI, and run `sudo apt update` to make sure that all of your packages are up to date.  
+
+7) Navigate to your home folder. *(`cd ~`)*
+
+8) Make sure you have *git* installed *(`sudo apt install git`)*  Then, `git clone https://github.com/Chew/DIRCord.git` 
```

Comment:
Remove the first bit and just keep `git clone` part. Git was already stated as a requirement before.